docs/sources/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/index.md

```diff
@@ -301,3 +305,41 @@ roles:
 ```
 
 - Or use [RBAC HTTP API]({{< relref "../../../../developers/http_api/access_control/#update-a-role" >}}).
+
+### Manage user permissions through teams
+
+In the scenario where you want users to have granted access by the team they belong to, we recommend to set users role to `No Basic Role` and let the team assignment assign the role instead.
+
+1. In Grafana, ensure the following configuration settings are enabled.
+   1. `users.auto_assign_org = true` this will automatically assign users to the default organization.
```

Comment:
How do these substeps render in the published docs? If they render as `1.`, I recommend manually changing these substeps to a., b., c. etc.

Reply:
I tried to render them with letters but the style was not the same as the numbers. I refactored it in order to show a configuration example instead.

Reply:
Looks good! Thank you!